### ISSUE TITLE
/multitool get <tool>

### ISSUE THREAD

**Issue by DukeVindzor** [TheDreamSanctuary/Multitool#6]
Command that gives player the item linked to the tool requested.

**Example:** /multitool get paintbrush
Should give the player a slimeball in inventory (if slimeball is the item associated with the paintbrush).

It should be done though the alias functionality so that player can write /multitool, /tool or maybe just /mt.

**Comment by AceDexter** [comment]
Do we need the 'get' part in the command? I guess we would if we want to add extra features to the /multitool command. Did you have any ideas on what more we could do with /multitool with a modifier?

**Comment by DukeVindzor** [comment]
Yes, I put up another issue kind of answering that. It is also better to have the possibility to add more commands if we need to in the future :)

**Comment by AceDexter** [comment]
Gotcha ^_^

**Comment by AceDexter** [comment]
Do we want the command to be /multitool get <tool> or /multitool give <tool> ? 

I personally prefer "give" but if "get" is better that's fine, just explain please =)

**Comment by DukeVindzor** [comment]
I added both actually :) This command is pretty much done, but I will need to wait until I have a tool for testing

**Comment by AceDexter** [comment]
@DukeVindzor Nice work! Have you pushed these changes yet?

**Comment by DukeVindzor** [comment]
No, I need to properly test it.

**Comment by DukeVindzor** [comment]
Added the extra functionality of "/mt get all" which gives all the tools the player has permission to get.
